r/tests/testthat/test-dplyr-filter.R

```diff
@@ -239,6 +239,14 @@ test_that("filter() with between()", {
       filter(between(chr, 1, 2)) %>%
       collect()
   )
+
+  # with namespacing
+  expect_error(
```

Comment:
Should this assert the error message? As in, this test would pass on master because it would error, but the error message would be different right?

Reply:
There are a bunch of tests in that chunk that do not assert the error message since it's being surfaced from C++. I've changed the test to a non-error one, since that might be more relevant for our use case. Done.